### PR TITLE
Language updates

### PR DESCRIPTION
**Pull request by filipviz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,35 +1,60 @@
...
+| `forge clean`         | Remove the build artifacts and cache directories.   |
 
-## Code coverage
-Run `yarn coverage`to display code coverage summary and generate an LCOV report
+To learn more, visit the [Foundry Book](https://book.getfoundry.sh/) docs.
 
-To display code coverage in VSCode:
-- You need to install the [coverage gutters extension (Ryan Luker)](https://marketplace.visualstudio.com/items?itemName=ryanluker.vscode-coverage-gutters) or any other extension handling LCOV reports
-- ctrl shift p > "Coverage Gutters: Display Coverage" (coverage are the colored markdown lines in the left gutter, after the line numbers)
+We recommend using [Juan Blanco's solidity extension](https://marketplace.visualstudio.com/items?itemName=JuanBlanco.solidity) for VSCode.
```
Comment (simon-something) [inline README.md]: Do we? I find hh ext overall better, including with foundry

File: src/JBSwapTerminal.sol [hunk 2]
```diff
@@ -59,33 +55,37 @@ contract JBSwapTerminal is JBPermissioned, Ownable, IJBTerminal, IJBPermitTermin
...
+    /// The 447 units should be substracted from the previous cost, but we just want to compare.
+    ///
+    /// Scenario 2: With packing (packing in 6 words):
+    /// `mem_cost = 134**2 / 512 + 3 * 134 = 437 units`
+    /// Achieving a 6 word representation by downcasting `projectId` to a `uint96`,
+    /// then packing it with the pool address,
+    /// then packing `(tokenIn, inIsNativeToken)` and `(tokenOut, outIsNativeToken)`.
+    /// We need to pack once (in `pay(...)`) and adding a mask/shifting at least 6 times (18 units) is more
+    /// expensive than the memory expansion cost.
+    struct SwapConfig {
```
Comment (simon-something) [inline src/JBSwapTerminal.sol]: I guess we can remove this

File: README.md (hunk 1 above)
Comment (filipviz) [reply README.md]: Great point. I'll update this in all the READMEs.

File: src/JBSwapTerminal.sol (hunk 2 above)
Comment (filipviz) [reply src/JBSwapTerminal.sol]: Hahaha yeah. Maybe as a fun fact in a doc somewhere. I'll leave it to you!

File: README.md [hunk 3]
```diff
@@ -1,35 +1,60 @@
...
+
+Or make the script executable and run:
+
+```bash
+./util.sh --help
+```
+
+## Terminals
+
+Juicebox projects can accept funds through one or more *terminals*, which can manage both inflows (via payments) and outflows (via redemptions). Terminals usually only accept one token for payments, but if a project has a swap terminal, the swap terminal can accept any token and swap it for tokens which the project *can* accept. After swapping, it redirects the payment to the primary terminal for the token it received.
```
Comment (simon-something) [inline README.md:56]: Maybe add emphasis on the diff classic (pay/redeem) terminal vs this one (pay only)?

**Review by mejango — APPROVED** [review]

**Pull request merged**